contribution-guide.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: ballerina-inner-page
+title: The Ballerina Contribution Guide
+permalink: /contribution-guide/
+---
+
+# The ballerina Contribution Guide
+
+Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is an open-source project under the Apache license and the work of hundreds of contributors.
+
+We appreciate your help!
+
+- [Getting engaged](#getting-engaged)
+- [Getting started](#getting-started)
+- [Obtaining the source code](#obtaining-the-source-code)
+- [Building the project](building-the-project)
+- [Setting up the developer environment](setting-up-the-developer-environment)
+    - [Setting up IntelliJ IDEA](#setting-up-intelliJ-idea)
+    - [Setting up Eclipse](#setting-up-eclipse)
+- [Contributing to Ballerina grammar](#contributing-to-ballerina-grammar)
+- [Submitting your contribution](#submitting-your-contribution)
+- [Accepting the Contributor License Agreement](#accepting-the-contributor-license-agreement)
+- [Proposing changes to Ballerina](#proposing-changes-to-ballerina)
+
+## Getting engaged
+
+- The [Ballerina-Dev Google Group](#https://groups.google.com/forum/#!forum/ballerina-dev) is the main Ballerina project discussion forum. Also, you can use [StackOverflow](#https://stackoverflow.com/questions/tagged/ballerina) for support, [Slack](#https://app.slack.com/client/T47EAEKB5/DF818FX9Q) for real-time communications.
```

Comment:
Seems links are not added properly. They are not visible in .md file. Check all the places.

Reply:
Changed as follows.

- Join the conversations at:

    - [Ballerina-Dev Google Group](https://groups.google.com/forum/#!forum/ballerina-dev): to join the developer team mailing list to discuss Ballerina roadmap, features, issues that are in progress etc.
    - [StackOverflow](https://stackoverflow.com/questions/tagged/ballerina): to get help with Ballerina. (Use the Ballerina tag for your questions there.) 
    - [Slack](https://app.slack.com/client/T47EAEKB5/DF818FX9Q): for real-time communications with the team and community.

Reply:
Added the icons.